presto-pinot/src/test/java/io/prestosql/pinot/util/TestingPinotCluster.java

```diff
@@ -0,0 +1,219 @@
+
+        server = new FixedHostPortGenericContainer<>(BASE_IMAGE)
+                .withNetwork(Network.SHARED)
+                .withClasspathResourceMapping("/pinot-server", "/var/pinot/server/config", BindMode.READ_ONLY)
+                .withEnv("JAVA_OPTS", "-Xms512m -Xmx512m -Dlog4j2.configurationFile=/opt/pinot/conf/pinot-server-log4j2.xml -Dplugins.dir=/opt/pinot/plugins")
+                .withCommand("StartServer", "-clusterName", "pinot", "-zkAddress", getZookeeperInternalHostPort(), "-configFileName", "/var/pinot/server/config/pinot-server.conf")
+                .withNetworkAliases("pinot-server", "localhost")
+                .withFixedExposedPort(SERVER_PORT, SERVER_PORT)
+                .withFixedExposedPort(SERVER_ADMIN_PORT, SERVER_ADMIN_PORT);
+    }
+
+    public void start()
+    {
+        zookeeper.start();
+        controller.start();
+        broker.start();
+        server.start();
+    }
+
+    @Override
+    public void close()
+    {
+        server.stop();
+        broker.stop();
+        controller.stop();
+        zookeeper.stop();
+    }
+
+    private static String getZookeeperInternalHostPort()
+    {
+        return format("%s:%s", ZOOKEEPER_INTERNAL_HOST, ZOOKEEPER_PORT);
+    }
+
+    public String getControllerConnectString()
+    {
+        return controller.getContainerIpAddress() + ":" + controller.getMappedPort(CONTROLLER_PORT);
+    }
+
+    public HostAndPort getBrokerHostAndPort()
+    {
+        return HostAndPort.fromParts(broker.getContainerIpAddress(), broker.getMappedPort(BROKER_PORT));
+    }
+
+    public HostAndPort getServerHostAndPort()
+    {
+        return HostAndPort.fromParts(server.getContainerIpAddress(), server.getMappedPort(SERVER_PORT));
+    }
+
+    public void createSchema(InputStream tableSchemaSpec, String tableName)
+            throws Exception
+    {
+        byte[] bytes = ByteStreams.toByteArray(tableSchemaSpec);
+        Request request = Request.builder().preparePost()
+                .setUri(getControllerUri("schemas"))
+                .setHeader(HttpHeaders.ACCEPT, APPLICATION_JSON)
+                .setHeader(HttpHeaders.CONTENT_TYPE, APPLICATION_JSON)
+                .setBodyGenerator(StaticBodyGenerator.createStaticBodyGenerator(bytes))
+                .build();
+
+        PinotSuccessResponse response = doWithRetries(() -> httpClient.execute(request, createJsonResponseHandler(PINOT_SUCCESS_RESPONSE_JSON_CODEC)), 10);
+        checkState(response.getStatus().equals(format("%s successfully added", tableName)), "Unexpected response: '%s'", response.getStatus());
+        verifySchema(tableName);
+    }
+
+    private URI getControllerUri(String path)
+    {
+        return URI.create(format("http://%s/%s", getControllerConnectString(), path));
+    }
+
+    public void verifySchema(String tableName)
+            throws Exception
+    {
+        Request request = Request.builder().prepareGet().setUri(getControllerUri("schemas"))
```

Comment:
`prepareGet()` is a static method on Request.Builder, so do this instead: `Request.Builder.prepareGet()`

This applies to all other usages of prepareXXX